haystack_experimental/components/builders/answer_builder.py

```diff
@@ -0,0 +1,200 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import re
+from typing import Any, Dict, List, Optional, Union
+
+from haystack import Document, component, logging
+from haystack.dataclasses.chat_message import ChatMessage
+
+from haystack_experimental.dataclasses import GeneratedAnswer
+
+logger = logging.getLogger(__name__)
+
+
+@component
+class AnswerBuilder:
```

Comment:
Sorry if I missed why, but could you briefly explain why AnswerBuilder needs to be included in Experimental?